src/ig/IGExporter.ts

```diff
@@ -251,7 +251,38 @@ export class IGExporter {
     const inputIndexMarkdownPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.md');
     const inputIndexXMLPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.xml');
     let generation: ImplementationGuideDefinitionPageGeneration = 'markdown';
-    if (existsSync(inputIndexMarkdownPageContentPath)) {
+    let filePath: string;
+    if (existsSync(inputIndexMarkdownPageContentPath)) filePath = inputIndexMarkdownPageContentPath;
+    if (existsSync(inputIndexXMLPageContentPath)) filePath = inputIndexXMLPageContentPath;
+    if (existsSync(inputIndexMarkdownPagesPath)) filePath = inputIndexMarkdownPagesPath;
+    if (existsSync(inputIndexXMLPagesPath)) filePath = inputIndexXMLPagesPath;
+
+    if (this.config.indexPageContent) {
+      ensureDirSync(pageContentExportPath);
+      logger.info('Generated index.md based on "indexPageContent" in config.yaml.');
+      const warning = warningBlock('<!-- index.md {% comment %}', '{% endcomment %} -->', [
+        'To change the contents of this file, edit the "indexPageContent" attribute in the tank config.yaml file',
+        'or provide your own index file in the ig-data/input/pagecontent or ig-data/input/pages folder.',
+        'See: https://build.fhir.org/ig/FHIR/ig-guidance/using-templates.html#root.input'
+      ]);
+      const outputPath = path.join(pageContentExportPath, 'index.md');
+      outputFileSync(outputPath, `${warning}${this.config.indexPageContent}`);
+      this.updateOutputLog(outputPath, [this.configPath], 'generated');
```

Comment:
Since below we have logs for when we copy `index.[md | xml]`, I think it would be fair here to have a log that we have generated `index.md`. And to be clear I mean a `logger` log, not a `updateOutputLog` log.

Reply:
There actually is one that I think fulfills this case, it's just up on line 262 rather than at the end. Is that log what you were thinking? If so, I can move it down to mirror the other cases.

Reply:
Oh yeah oops, no need to move!

Reply:
Now I want them to match though, so I'm going to move it.